config/operator.yaml

```diff
@@ -59,8 +59,33 @@ spec:
             periodSeconds: 1
             successThreshold: 1
             timeoutSeconds: 15
+          securityContext:
+            runAsUser: 65532
+            runAsGroup: 65532
+            runAsNonRoot: true
+            readOnlyRootFilesystem: true
+            seccompProfile:
+              type: RuntimeDefault
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
           terminationMessagePolicy: FallbackToLogsOnError
+          volumeMounts:
+            - mountPath: /tmp
+              name: scratch
+              subPath: tmp
+      securityContext:
+        runAsUser: 65532
```

Comment:
I set them in both because the pod context are the defaults that get applied when there are sidecar containers injected with admission controllers (unless the sidecar overrides those). So this is a kind of belt-and-suspenders approach to cover all of our bases

Reply:
That's reasonable, thanks